web_src/css/base.css

```diff
@@ -1194,3 +1194,7 @@ the "!important" is necessary to override Fomantic UI menu item styles, meanwhil
 .svg.octicon-file-directory-symlink {
   color: var(--color-secondary-dark-7);
 }
+
+span:has(> svg:only-child) {
+  display: contents;
+}
```

Comment:
I don't think it could be a general style.

What would you expect for `<span><svg/>text</span>`?

Reply:
I think that elements like `<span><svg/></span>` should also use the contents layout, and this change can indeed cover it.

Reply:
But it's not right, it just depends the parent's "flex" layout.

If the parent doesn't have flex layout, this svg still mis-aligns ..........

I don't think it is a proper approach.

Reply:
OK, thanks for the reminder. Then I'll try another solution.